mitiq/lre/tests/test_layerwise_folding.py

```diff
@@ -215,11 +215,11 @@ def test_layers_with_chunking():
         ),
     ],
 )
-def test_get_scale_factor_vectors_no_chunking(
+def testget_scale_factor_vectors_no_chunking(
```

Comment:
Why did the underscore here go away?

Reply:
I used search and replace to make a private function public. This underscore was also affected by that action. 

Will change back.